src/components/complete-profile-second/work-form/select-options.ts

```diff
@@ -0,0 +1,37 @@
+import { useLocales } from 'src/locales';
+
+export type SelectWorkTypes = 'Hospital' | 'Clinic' | 'Agency' | 'Other';
+
+type SelectOptions = {
+  value: SelectWorkTypes;
+  label: string;
+};
+
+type WorkTypes = {
```

Comment:
It would be better to extract these types to a separate 'types.ts' file because here there are quite a few of them

Reply:
fixed